styles/Stars.module.css

```diff
@@ -7,9 +7,15 @@
   width: 45px;
   overflow: unset;
   margin-right: 15px;
+  color: gray;
 }
 .star:hover {
-  opacity: 0.5;
+  cursor: pointer;
+  color: #a98d00;
+}
+
+.selected {
```

Comment:
Not sure how I feel about the new highlight color. For example, how many stars are selected in this screenshot?
<img width="451" alt="Screenshot 2023-11-20 at 1 12 15 PM" src="https://github.com/ibi-group/survey/assets/86619099/953b4f5e-005c-4cf7-a142-fcb2d32a4241">
vs this one?
<img width="440" alt="Screenshot 2023-11-20 at 1 11 16 PM" src="https://github.com/ibi-group/survey/assets/86619099/c13ba816-79e9-4471-b271-fb6e9c5cfe87">


I think a better solution might be to use outlines?

Reply:
How were you able to get to the state in your second screenshot? I see how that's an issue but I can't reproduce it [here](https://st-survey-git-improve-star-hover-ibi-group.vercel.app/). I think it's worth I try adding outlines to close https://github.com/ibi-group/survey/issues/18. What do you think of a light outline on hover and a thicker outline on select? I think it could be productive if I bring that up on the tech call so I don't make changes that don't make sense.